### DISCUSSION TITLE
Can we integrate the GrapesJS Studio?

### DISCUSSION THREAD

**Discussion by krishnabose02** [GrapesJS/grapesjs#5365] [General]
Hi, I would like to know if GrapesJS Studio is currently integrable into a custom project for trying it out.
Also, i would like to understand the roadmap with this (whether its a successor to original editor, or is it going to be a separate project or going to be a premium alternative to existing editor)
I have found very limited information elsewhere, so it would be helpful if you can guide me regarding the same.

**Comment by chrisedington** [comment]
@artf it's been a while since I've been here but would love to know more about Studio too

**Comment by artf** [comment]
GrapesJS Studio it's an application built on top of the GrapesJS framework. You can see it as a separate project, so it's not going to "replace" the framework.

Currently, I don't provide a way right now to integrate it into external projects but it's something I'm planning to do.

**Reply by krishnabose02** [reply]
@artf Thanks a lot. Would look forward to it.

**Reply by chrisedington** [reply]
Sounds very interesting 👌

**Comment by glaprida** [comment]
Hey @artf Is there any chance you share the unminified js code you used in GrapesJS Studio. It would be really neat just to go over it and learn how you actually use and customize GrapesJS. I think I would learn a lot.

**Reply by artf** [reply]
not yet, but I'm mainly using https://github.com/GrapesJS/react
